### PR TITLE
Deploy to heroku

### PR DESCRIPTION
**Pull request by whaley-ethan**
Made changes neccessary to deploy to heroku, including hardcoded http header

### CODE REVIEW

File: src/echo_server/core.clj [hunk 1]
```diff
@@ -9,7 +10,7 @@
     (. writer write line 0 (count line))))
 
 (defn write-hello-world [out]
-    (sock-write out (str "hello world!"))
+    (sock-write out (str "HTTP/1.1 200 OK\r\nContent-Type:text/html\r\nContent-Length:14\r\n\r\nhello world!"))
```
Comment (jchung722) [inline src/echo_server/core.clj]: Shouldn't `Content-Length` be 12?

Comment (whaley-ethan) [reply src/echo_server/core.clj]: yes. apparently i miscounted.

**Review by jchung722 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,36 +1,28 @@
...
-### Bugs
+or you can compile into an uberjar using
 
-...
+`lein uberjar`
 
-### Any Other Sections
-### That You Think
-### Might be Useful
+The uberjar command creates 2 uberjars, in the `/target` directory the `standalone` uberjar is suitable for deployment and includes all needed dependincies
```
Comment (whaley-ethan) [inline README.md:22]: add Deployment section

File: project.clj [hunk 3]
```diff
@@ -1,8 +1,17 @@
-(defproject echo-server "0.1.0-SNAPSHOT"
+(defproject echo_server "0.1.0-SNAPSHOT"
   :description "FIXME: write description"
   :url "http://example.com/FIXME"
   :license {:name "EPL-2.0 OR GPL-2.0-or-later WITH Classpath-exception-2.0"
             :url "https://www.eclipse.org/legal/epl-2.0/"}
   :dependencies [[org.clojure/clojure "1.10.1"]]
-  :main "echo-server.core/main"
-  :profiles {:uberjar {:aot :all}})
+  :plugins [[lein-heroku "0.5.3"]]
```
Comment (whaley-ethan) [inline project.clj]: remove plugin -- not being used

Comment (whaley-ethan) [reply project.clj]: done

**Review by ericmeyer — APPROVED** [review]

**Review by elijahsullivan — APPROVED** [review]

**Pull request merged**